src/bilder/images/edxapp_v2/files/docker-compose.yaml.tmpl

```diff
@@ -12,7 +12,7 @@ services:
     command: ["1000", "/openedx/data", "/openedx/media"]
     restart: on-failure
     volumes:
-    - /opt/data:/openedx/data
```

Comment:
Is there anything else under the /opt/data directory besides the logs that the lms/cms would need access to?